modules/govuk_jenkins/templates/jobs/check_content_consistency.yaml.erb

```diff
@@ -0,0 +1,87 @@
+---
+- scm:
+    name: check-content-consistency
+    scm:
+      - git:
+          url: git@github.gds:gds/router-data.git
+          branches:
+            - master
+- job:
+    name: Check_Content_Consistency
+    display-name: Check_Content_Consistency
+    project-type: freestyle
+    description: |
+      This job checks for consistency between the router-api, content-store
+      and publishing-api.
+    scm:
+      - check-content-consistency
+    logrotate:
+      numToKeep: 10
+    triggers:
+        - timed: 'H 5 * * *'
+    builders:
+      - shell: |
+          #!/bin/bash
+          set -eu
+
+          cd "${WORKSPACE}"
+
+          function run_on_router_backend() {
+            ssh deploy@router-backend-1.backend.<%= @app_domain -%> "$@"
+          }
+
+          function run_on_draft_cache() {
+            ssh deploy@draft-cache-1.backend.<%= @app_domain -%> "$@"
+          }
+
+          function run_on_content_store() {
+            ssh deploy@content-store-1.backend.<%= @app_domain -%> "$@"
+          }
+
+          function run_on_draft_content_store() {
+            ssh deploy@draft-content-store-1.backend.<%= @app_domain -%> "$@"
+          }
+
+          function run_on_backend() {
+            ssh deploy@backend-1.backend.<%= @app_domain -%> "$@"
```

Comment:
As a rule we tend to suggest that you use a randomly selected (available) machine from that machine class ie choose something from the results of `govuk_node_list -c backend` rather than always choosing `backend-1`, unless there a specific reason it needs to be tied to that machine. If we need to take down a machine for maintenance and/or remove it from Puppet this would mean that your job would still run without failing.